Roboto-min.designspace

```diff
@@ -1,252 +1,264 @@
 <?xml version='1.0' encoding='utf-8'?>
 <designspace format="3">
     <axes>
-        <axis default="-0.1" maximum="1.125" minimum="-1" name="weight" tag="wght">
+        <axis default="400" maximum="900" minimum="250" name="weight" tag="wght">
             <labelname xml:lang="en">Weight</labelname>
+            <map input="250" output="-1" />
+            <map input="300" output="-0.550000" />
+            <map input="400" output="-0.1" />
+            <map input="500" output="0.350000" />
+            <map input="700" output="0.730000" />
+            <map input="900" output="1.125" />
         </axis>
         <axis default="100" maximum="100" minimum="75" name="width" tag="wdth">
             <labelname xml:lang="en">Width</labelname>
+            <map input="75" output="-1" />
+            <map input="87.5" output="0" />
+            <map input="100" output="1" />
         </axis>
         <axis default="0" maximum="12" minimum="0" name="slant" tag="slnt">
             <labelname xml:lang="en">Slant</labelname>
+            <map input="0" output="-1" />
+            <map input="6" output="0" />
+            <map input="12" output="1" />
```

Comment:
As I said, the user-space coordinates for the `slnt` axis (what gets in the map's "input" attribute) must be negative in a typical right-leaning oblique design like Roboto

https://docs.microsoft.com/en-us/typography/opentype/spec/dvaraxistag_slnt

Also, the intermediate mapping `<map input="6" output="0" />` is redundant in this case, since it's exactly at the middle, so you can remove it.